### DISCUSSION TITLE
Question 66: Could you please point out the Site Jurisdiction field & provide an island refresher? [RB-INS]

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#1073] [Q&A] [answered]
Description: A few years ago, we made decisions about how to parent the Caribbean islands and countries (https://github.com/fieldmuseum/EMu-Documentation/issues/90).  

Could you please point out where to find the SitJurisdiction field?
- E.g. SitJurisdiction = United States
- PolPD1 [LocCountry_tab] = Puerto Rico

Would you also please give a refresher on how to make records for sites that are on islands? It’s been a few years. We especially need clarification for records where the continent of the country with jurisdiction is different from the island’s continent and how to construct records for the islands with multiple countries on them (Borneo or Hispaniola). Thank you!

**Comment by rondlg** [comment, marked as answer]
Response: Jurisdiction is on the site tab. An example is in the Sites workflow document [here](https://docs.google.com/document/d/1YA87zYvahTko2McaCWmTofg7XBdjKAbfZjJSG6KfS6Q/edit#heading=h.w64r7k9g7w8f)

@rbanasiak

**Reply by rbanasiak** [reply]
Hi Sharon....I think you meant to send this to Becka in Insects.

**Reply by rondlg** [reply]
OOps - Thanks

@rbaquiran I meant to tag you!

**Comment by rbaquiran** [comment]
Thank you, Sharon!

**Comment by rondlg** [comment]
Discussed at https://github.com/fieldmuseum/EMu-Documentation/discussions/1084. Standard agreed.
